perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1420,4 +1420,15 @@
 	 * @throws UserNotExistsException if there is no such user
 	 */
 	void updateSponsorshipValidity(PerunSession sess, Member sponsoredMember, User sponsor, LocalDate newValidity) throws PrivilegeException, SponsorshipDoesNotExistException, MemberNotExistsException, UserNotExistsException;
+
+
+	/**
+	 * Returns rich members with specific attributes by their IDs.
```

Comment:
The name is a little confusing. It looks like you would like to get attributes by their IDs.

Reply:
Fixed